_build/generate-phpdoc.sh

```diff
@@ -15,7 +15,7 @@ echo
 
 # generate phpdoc
```

Comment:
With the addition of the code sniffer `.xml`, should we also have a `.phpdoc.xml` file in the root instead of hiding the ignore list inside the script?